src/main/java/com/iexec/core/replicate/ReplicatesService.java

```diff
@@ -346,10 +360,24 @@ private boolean isTaskStatusFailOnChain(String chainTaskId, String walletAddress
         }
     }
 
+    boolean isPublicResult(String chainTaskId, Integer chainId) {
```

Comment:
Can we move `isPublicResult` to common? (We're gonna need it for the seperate Result Repo)

Reply:
right, moved to common.